l10n_nl_bsn/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================================
+Burgerservicenummer (BSN) for Partners
+======================================
+
+This module adds the BSN (Burgerservicenummer) field on partner forms.
+
+The field is visible when the flag is_company is false.
```

Comment:
This line is not true anymore, as flag is_company was removed in v12. The field is visible, instead, when "Individual" radio button is checked.

Reply:
This is already fixed in file `readme/DESCRIPTION.rst` (https://github.com/OCA/l10n-netherlands/pull/199/files#diff-351c10adac39025dad8b1371e573db34R3). The `README.rst` will be automatically overwritten.